pkgs/data/fonts/kanji-stroke-order-font/default.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, fetchzip }:
+
+let
+  version = "v4.002";
+in fetchzip {
+  name = "kanji-stroke-order-font-${version}";
+
+  url = "https://sites.google.com/site/nihilistorguk/KanjiStrokeOrders_${version}.zip?attredirects=0";
```

Please remove the `v` from `version`, so that it follows the [styleguide](https://nixos.org/nixpkgs/manual/#sec-package-naming)

```suggestion
  url = "https://sites.google.com/site/nihilistorguk/KanjiStrokeOrders_v${version}.zip?attredirects=0";
```